pnpm-lock.yaml

```diff
@@ -619,6 +622,15 @@ packages:
     cpu: [arm64]
     os: [android]
     requiresBuild: true
+    dev: true
+    optional: true
+
+  /@esbuild/android-arm64@0.18.11:
```

Comment:
most of these changes should go away if you merge in the latest changes from `master`